x-pack/test/security_solution_cypress/cypress/tasks/prebuilt_rules.ts

```diff
@@ -142,3 +236,306 @@ export const interceptUpgradeRequestToFail = (rules: Array<typeof SAMPLE_PREBUIL
     delay: 500, // Add delay to give Cypress time to find the loading spinner
   }).as('updatePrebuiltRules');
 };
+
+export const assertRuleInstallSuccessToastShown = (rules: Array<typeof SAMPLE_PREBUILT_RULE>) => {
+  const rulesString = rules.length > 1 ? 'rules' : 'rule';
+  cy.get(TOASTER)
+    .should('be.visible')
+    .should('have.text', `${rules.length} ${rulesString} installed successfully.`);
+};
+
+export const assertRuleUpgradeSuccessToastShown = (rules: Array<typeof SAMPLE_PREBUILT_RULE>) => {
+  const rulesString = rules.length > 1 ? 'rules' : 'rule';
+  cy.get(TOASTER)
+    .should('be.visible')
+    .should('have.text', `${rules.length} ${rulesString} updated successfully.`);
+};
+
+export const assertRulePresentInInstalledRulesTable = (ruleName: string) => {
+  cy.get(RULES_MANAGEMENT_TABLE).contains(ruleName);
+};
+
+export const assertRuleNotPresentInAddPrebuiltRulesTable = (ruleName: string) => {
+  cy.get(ADD_ELASTIC_RULES_TABLE).contains(ruleName).should('not.exist');
+};
+
+export const assertRuleNotPresentInRuleUpdatesTable = (ruleName: string) => {
+  cy.get(RULES_UPDATES_TABLE).contains(ruleName).should('not.exist');
+};
+
+export const openRuleInstallPreview = (ruleName: string) => {
+  cy.contains(ruleName).click();
+  cy.get(INSTALL_PREBUILT_RULE_PREVIEW).should('be.visible');
+};
+
+export const openRuleUpdatePreview = (ruleName: string) => {
+  cy.contains(ruleName).click();
+  cy.get(UPDATE_PREBUILT_RULE_PREVIEW).should('be.visible');
+};
+
+export const closeRulePreview = () => {
+  cy.get(FLYOUT_CLOSE_BTN).click();
+  cy.get(INSTALL_PREBUILT_RULE_PREVIEW).should('not.exist');
+};
+
+export const assertCommonPropertiesDisplayedCorrectly = (
```

Comment:
nit: Suffix part `DisplayedCorrectly` looks inconsistent with the functions below as functions use suffix `Displayed`. Maybe simplify and name functions with suffix `Shown` (less letters) like `assertCommonPropertiesShown`? As specific properties a passed `Correctly` is implied.